apisix/stream/router/ip_port.lua

```diff
@@ -33,14 +33,30 @@ local _M = {version = 0.1}
 
 local function match_addrs(route, vars)
     -- todo: use resty-ipmatcher to support multiple ip address
-    if route.value.remote_addr and
-       route.value.remote_addr ~= vars.remote_addr then
-        return false
+    if route.value.remote_addr then
```

Comment:
Better to put the init in the `create_router` instead of the match

Reply:
Sorry I don't get your means, you mean remove the match function?